src/dotnet/commands/dotnet-build/BuildCommandParser.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class BuildCommandParser
+    {
+        public static Command Build() =>
+            Create.Command("build",
+                           ".NET Builder",
+                           CommonOptions.HelpOption(),
+                           Create.Option("-o|--output",
+                                         "Output directory in which to place built artifacts.",
+                                         Accept.ExactlyOneArgument
+                                               .With(name: "OUTPUT_DIR")),
+                           Create.Option("-f|--framework",
+                                         "Target framework to build for. The target framework has to be specified in the project file.",
+                                         Accept.AnyOneOf(Suggest.TargetFrameworksFromProjectFile)),
+                           Create.Option("-r|--runtime",
+                                         "Target runtime to build for. The default is to build a portable application.",
+                                         Accept.AnyOneOf(Suggest.RunTimesFromProjectFile)),
```

Comment:
No but we can change it easily. There are two methods (with overloads) you'll see here:

* `Accept.AnyOneOf` will result in a parse error if the provided value is not in the set, and
* `Accept.WithSuggestionsFrom` will provide completions from the provided values, but other input is also allowed.